packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -302,6 +304,7 @@ export const messages = {
   error_overwrite_prompt: 'Error checking workspace for existing components',
   error_no_scratch_def:
     'No scratch definition files found. These files must be in the "config" folder and have file names that end with "-scratch-def.json". See [Scratch Org Definition File](https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_scratch_orgs_def_file.htm) for help.',
+  error_function_error: 'Function creation error',
```

Comment:
Can we provide more details about how to get out of the jam?

Reply:
I'm handling error handling as my next story. This message can be removed for now. The next story will have some pre checks and how to resolve them.